dockerfiles/PySparkContainerDockerfile

```diff
@@ -6,7 +6,6 @@ FROM ${REGISTRY}/${RPC_VERSION}-rpc:${CODE_VERSION}
 LABEL maintainer="Dan Crankshaw <dscrankshaw@gmail.com>"
 
 RUN mkdir -p /usr/share/man/man1 && \
-    (echo "deb http://http.debian.net/debian jessie-backports main" > /etc/apt/sources.list.d/backports.list) && \
     apt-get update -qq -y && \
```

Comment:
You don't need this update if you're not adding anything to the apt packages list.
```suggestion
   
```

Reply:
I missed that the base image(${REGISTRY}/${RPC_VERSION}-rpc:${CODE_VERSION}) has already the updated apt packages list. I will delete the code right now. Thank you, @RehanSD .